content/en/docs/concepts/policy/limit-range.md

```diff
@@ -352,15 +353,15 @@ Type        Resource  Min  Max  Default Request  Default Limit  Max Limit/Reques
 Pod         memory    -    -    -                -              2
 ```
 
-Let's create a pod with `requests.memory=100Mi` and `limits.memory=300Mi`:
+Create a pod with `requests.memory=100Mi` and `limits.memory=300Mi`:
 
 {{< codenew file="admin/resource/limit-range-pod-3.yaml" >}}
 
 ```shell
 kubectl apply -f https://k8s.io/examples/admin/resource/limit-range-pod-3.yaml
 ```
 
-The pod creation failed as the ratio here (`3`) is greater than the enforced limit (`2`) in `limit-memory-ratio-pod` LimitRange
+The pod creation failed as the ratio here (`3`) is greater than the enforced limit (`2`) in `limit-memory-ratio-pod` LimitRange:
 
 ```shell
```

Comment:
Aside: this is output, it shouldn't be marked as shell.

@danielhelfand it's up to you if you want to fix that or leave it to a different PR.

Reply:
I have updated it.